src/Uno.UWP/Storage/Streams/IBufferByteAccess.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Windows.Storage.Streams
+{
+	public interface IBufferByteAccess
```

Comment:
What's the point of this interface? There is only one implementor anyway (Buffer).